common/lib/xmodule/xmodule/course_module.py

```diff
@@ -386,6 +386,18 @@ class CourseFields(object):
         ),
         scope=Scope.settings
     )
+    enable_ccx = Boolean(
+        # Translators: Custom Courses for edX (CCX) is an edX feature for re-using course content. CCX Coach is
+        # a role created by a course Instructor to enable a person (the "Coach") to manage the custom course for
+        # his students.
+        display_name=_("Enable CCX"),
+        help=_(
```

Comment:
Why did you delete the translators comment for this string? Like I said, EVERY string needs a translators comment. Please re-read what I wrote and add a translators comment for the "help" string *in addition to* the display_name string. I even gave you the translators comment to add.

This is good but translators may not translate strings sequentially or different translators may translate different strings. So every string that mentions CCX should explain what CCX is.

It is sufficient to say:
`Custom Courses for edX (CCX) is an edX feature for re-using course content. CCX Coach is a role created by a course Instructor to enable a person (the "Coach") to manage the custom course for his students.`